pkg/cmd/start/start_pipeline.go

```diff
@@ -401,3 +422,30 @@ func (o *StartPipelineOptions) startJenkinsJob(name string) error {
 		time.Sleep(time.Second)
 	}
 }
+
+func (o *StartPipelineOptions) prowToMetaPipelinePullRef(sourceURL string, prowPullRef *prow.PullRefs) metapipeline.PullRef {
```

Comment:
These methods seems to be duplicated in pipelinerunner. Shouldn't we move them in `prow` package since they seems related to it?

Reply:
yes, I was thinking about this. I went for duplication for now, for two reasons. I did not want to depend on another package (even though the prow package is probably safe) and secondly because eventually we will move away from Prow. Once we fully embrace lighthouse it does not make sense anymore to use a Prow pull ref. 

I can take another look.

Reply:
fair enough for now. Thanks!

Reply:
Actually I could remove the duplication. I don't really need to go via a Prow pull ref, I can create the meta pipeline one directly.